dask/array/core.py

```diff
@@ -2398,7 +2398,7 @@ def min(self, axis=None, keepdims=False, split_every=None, out=None):
     def max(self, axis=None, keepdims=False, split_every=None, out=None):
         from dask.array.reductions import max
 
-        return max(self, axis=axis, keepdims=keepdims, split_every=split_every, out=out)
+        return max(self, axis=axis, keepdims=keepdims, split_every=split_every, out=out, maxtype=chunk.max)
```

Comment:
Included the maxtype argument to allow chunk.max to be used in cases in which an empty array is expected to be returned (e.g., some searchsorted method cases), which is not possible with chunk.max_wrapped.